src/backend/joanie/tests/core/test_utils_image_to_base64.py

```diff
@@ -13,6 +13,9 @@
 )
 
 
+@override_settings(
+    STORAGES={"default": {"BACKEND": "django.core.files.storage.FileSystemStorage"}}
+)
```

Comment:
Maybe targeting only the `path` test could be better ?

Reply:
Sorry, I'm not sure to get your idea.